budget-api/src/validators/validation-helpers.ts

```diff
@@ -128,7 +128,7 @@ export class ValidationHelpers {
     target: ValidationTarget,
     keys: string[]
   ): ValidationChain {
-    return this.getChecker(target, paramName).custom((value) => {
+    return this.getChecker(target, paramName).custom((value: KeyValueType) => {
```

Comment:
We can try and make this method generic (`validateObjectKeys<T>`) and then use the generic type in the `custom((value: T) => {...}` lambda.